File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task124_PERPLEXITY_llama-3-sonar-large-32k-chat.java
public class SensitiveData {
    private String data;

    public SensitiveData(String data) {
        this.data = data;
    }

    public String getData() {
        return data;
    }

    public static void main(String[] args) {
        SensitiveData data = new SensitiveData("1234-5678-9012-3456");
        System.out.println(data.getData());
    }
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task124_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <string>

class SensitiveData {
private:
    std::string data;

public:
    SensitiveData(std::string data) {
        this->data = data;
    }

    std::string getData() {
        return data;
    }
};

int main() {
    SensitiveData data("1234-5678-9012-3456");
    std::cout << data.getData() << std::endl;
    return 0;
}